client/app/services/custom-button/custom-button.html

```diff
@@ -1,12 +1,15 @@
 <span class="pull-left" ng-repeat="button in vm.customActions.buttons">
-  <button class="btn btn-default custom-button-action"
-          ng-if="vm.hasRequiredRole(button)"
-          ng-click="vm.invokeCustomAction(button)"
-          uib-tooltip="{{button.description}}"
-          tooltip-placement="top">
-    <i ng-class="button.options.button_icon" ng-style="{color: button.options.button_color }"></i>
-    {{ button.name }}
-  </button>
+  <span uib-tooltip="{{(button.enabled? button.description: button.disabled_text)}}"
```

Comment:
just curious, any reason why the tooltip wasn't applied directly to the button? suspicious of this new `<span>`

Reply:
It was addd because buttons in a disabled state do not show a tooltip. It has something to do with the cursor css.  I researched different solutions and the easiest/less hacky solution was to wrap the element in a simple span.